packages/valory/skills/market_creation_manager_abci/behaviours.py

```diff
@@ -659,7 +659,8 @@ def _collect_latest_open_markets(
         self, openingTimestamp_gte: int, openingTimestamp_lte: int
     ) -> Generator[None, None, Dict[str, Any]]:
         """Collect FPMM from subgraph."""
-        creator = self.params.approve_market_creator
+        creator = self.synchronized_data.safe_contract_address.lower()
+        self.context.logger.info(f"_collect_latest_open_markets {creator=}")
```

Comment:
```suggestion
        self.context.logger.debug(f"_collect_latest_open_markets {creator=}")
```

Reply:
I will delete this line, as it's contained in other parts of the logs.